api/v1beta1/azurecluster_validation_test.go

```diff
@@ -465,6 +465,9 @@ func TestResourceGroupValid(t *testing.T) {
 		err := validateResourceGroup(testCase.resourceGroup,
 			field.NewPath("spec").Child("networkSpec").Child("vnet").Child("resourceGroup"))
 		g.Expect(err).NotTo(HaveOccurred())
+		err = validateResourceGroup(testCase.resourceGroup,
+			field.NewPath("spec").Child("networkSpec").Child("privateDNSZoneResourceGroup"))
+		g.Expect(err).NotTo(HaveOccurred())
```

Comment:
These changes won't be needed as we just want to test the new function `validatePrivateDNSZoneResourceGroup`

Reply:
Yeah this was just an additional test case. I have removed it now.